app/Models/LandingPage.php

```diff
@@ -192,13 +192,13 @@ class LandingPage extends AbstractModel implements Sortable
             'default' => [
                 [
                     'name' => 'default',
-                    'ratio' => 1,
+                    'ratio' => 16 / 9,
```

Comment:
On desktop, they're gonna want the whole image to display as Alexis designed it. It's wonky, but use `2267 / 742` as the aspect ratio here as that's what the original image is.